Sources/MongoSwift/MongoClient.swift

```diff
@@ -1,9 +1,12 @@
 import Foundation
 import mongoc
 
-/// Options to use when creating a `MongoClient`.
+/// Options to use when creating a `MongoClient`. For fields that are also included in the connection string URI
+/// (ie. readConcern, writeConcern, readPreference, retryWrites), the values specified here will override the ones
+/// specified in the URI. If they are not specified here or in the URI, the server's default read concern will be used.
 public struct ClientOptions: CodingStrategyProvider, Decodable {
-    /// Determines whether the client should retry supported write operations.
+    /// Determines whether the client should retry supported write operations. Explicitly setting retry writes to `nil`
+    /// is the same as not specifying one at all.
```

Comment:
instead of including "Explicitly setting ..." in every option's docstring, it would be easier to just include it (maybe in combination with something like the updated string I mentioned below) in the docstring for the options struct itself.